src/main/java/pl/edu/mimuw/BigInt.java

```diff
@@ -27,16 +44,80 @@ public BigInt neg() {
   }
 
   public BigInt add(BigInt other) {
-    throw new IllegalStateException("TODO task 2: implement addition");
+    BigInteger sum;
```

Comment:
bardzo nieładne cwaniactwo xd zdecydowanie nie o to chodziło i wszyscy napisali to na tablicach, tak jak należało. Proszę nie rób tak w przyszłości obetnę punkty (implementacja times Cie troche ratuje ;) )